xtask/src/run.rs

```diff
@@ -29,12 +28,6 @@ enum Environment {
         #[clap(long)]
         cache_dir: PathBuf,
 
-        /// The Github API token to use if network requests to Github are made.
-        ///
-        /// This may be required if Github rate limits are exceeded.
-        #[clap(long)]
-        github_api_token: Option<String>,
```

Comment:
This served a purpose. We frequently were getting rate limited in CI. Please reinstate it.